dotcom-rendering/src/palette.ts

```diff
@@ -2877,6 +2877,12 @@ const syndicationButtonBorder: PaletteFunction = ({ design, theme }) => {
 	}
 };
 
+const appsEpicBackgroundLight = () => sourcePalette.neutral[97];
+const appsEpicBackgroundDark = () => sourcePalette.neutral[20];
+
+const appsEpicBorderLight = () => sourcePalette.brandAlt[400];
+const appsEpicBorderDark = () => sourcePalette.brandAlt[200];
```

Comment:
If you desire, these can still be `PaletteFunction`s, e.g.:

```suggestion
const appsEpicBorderDark: PaletteFunction = () => sourcePalette.brandAlt[200];
```